### ISSUE TITLE
Pretrained networks on what data?

### ISSUE THREAD

**Issue by anuragranj** [ClementPinard/SfmLearner-Pytorch#20]
Hi, 
Did you use cityscapes+kitti, or only kitti for the available pretrained networks.

**Comment by anuragranj** [comment]
Thanks.

**Comment by shahabty** [comment]
Hello @ClementPinard , 
Could you please let me know how many epochs does it take for Dispnet to converge?

Thanks,

**Comment by ClementPinard** [comment]
Hello, see the discussion here #32
Basically, it really depends on your smooth loss, but the way it's coded now, 200k iterations is a good estimate.

**Comment by anuragranj** [comment]
The Google drive in readme has tensorboard graps. It takes around 150-200 epochs, to my knowledge.
